### PR TITLE
Feat: support values file select

### PR DESCRIPTION
**Pull request by wangyikewxgm**
support values file select

Fixes https://github.com/kubevela/kubevela/issues/3909

Demo:
```yaml
cat <<EOF | vela up -f -
apiVersion: core.oam.dev/v1beta1
kind: Application
metadata:
  name: hello-kubernetes
spec:
  components:
    - name: hello-kubernetes
      type: helm
      properties:
        repoType: "helm"
        url: "https://wangyikewxgm.github.io/my-charts/"
        chart: "hello-kubernetes-chart"
        version: "0.1.0"
EOF
```

```shelll
vela port-forward hello-kubernetes
```
You will see

image

Then:
```yaml
apiVersion: core.oam.dev/v1beta1
kind: Application
metadata:
  name: hello-kubernetes
spec:
  components:
    - name: hello-kubernetes
      type: helm
      properties:
        repoType: "helm"
        url: "https://wangyikewxgm.github.io/my-charts/"
        chart: "hello-kubernetes-chart"
        version: "0.1.0"
        valuesFiles:
          - "values-production.yaml"
EOF
```

Will see:

image

Signed-off-by: 楚岳 <wangyike.wyk@alibaba-inc.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [x] Updated/Added any relevant [documentation] and [examples].
- [x] Unit/E2E Tests added.

### CODE REVIEW

File: addons/fluxcd/definitions/helm-release-def.yaml [hunk 1]
```diff
@@ -137,6 +140,8 @@ spec:
         		// +usage=The bucket region, optional
         		region?: string
         	}
+            // +usage=Alternative list of values files to use as the chart values (values.yamlis not included by default), expected to be a relative path in the SourceRef.Values files are merged in the order of this list with the last file overriding the first.
+            valuesFiles: [...string]
```
Comment (wonderflow) [inline addons/fluxcd/definitions/helm-release-def.yaml:144]: Is this field optional?

Comment (wangyikewxgm) [reply addons/fluxcd/definitions/helm-release-def.yaml]: Yes,fixed.

**Review by wonderflow — DISMISSED** [review]

File: addons/fluxcd/definitions/helm-release-def.yaml [hunk 2]
```diff
@@ -137,6 +140,8 @@ spec:
         		// +usage=The bucket region, optional
         		region?: string
         	}
+            // +usage=Alternative list of values files to use as the chart values (values.yamlis not included by default), expected to be a relative path in the SourceRef.Values files are merged in the order of this list with the last file overriding the first.
+            valuesFiles?: [...string]
```
Comment (barnettZQG) [inline addons/fluxcd/definitions/helm-release-def.yaml:144]: If set multiple files, what mean?

Comment (barnettZQG) [reply addons/fluxcd/definitions/helm-release-def.yaml:144]: Advice write the version changes in the readme

Comment (wangyikewxgm) [reply addons/fluxcd/definitions/helm-release-def.yaml:144]: as the usage tells. The last file will override the first.

Comment (wangyikewxgm) [reply addons/fluxcd/definitions/helm-release-def.yaml:144]: > Advice write the version changes in the readme

Every changes add a change log will let readme too large that contains too much versioned infos. A readme belong to a special version of addon. IMO no need contain other versions' info.

File: addons/fluxcd/definitions/helm-release-def.yaml [hunk 3]
```diff
@@ -137,6 +140,8 @@ spec:
         		// +usage=The bucket region, optional
         		region?: string
         	}
+            // +usage=Alternative list of values files to use as the chart values (values.yamlis not included by default), expected to be a relative path in the SourceRef.Values files are merged in the order of this list with the last file overriding the first.
```
Comment (barnettZQG) [inline addons/fluxcd/definitions/helm-release-def.yaml:143]:
```suggestion
            // +usage=Alternative list of values files to use as the chart values (values.yaml not included by default), expected to be a relative path in the SourceRef.Values files are merged in the order of this list with the last file overriding the first.
```

**Review by wonderflow — APPROVED** [review]

**Pull request merged**